package.json

```diff
@@ -8,11 +8,11 @@
     "docker:start": "docker-compose run windbreaker",
     "docker:stop": "docker-compose down",
     "pretest": "npm run lint",
-    "test": "docker-compose run test",
+    "test": "docker-compose run windbreaker npm run test:docker",
```

Comment:
Can we just use `docker-compose up` in both places now that `test` isn't a separate service listed in the compose file?

Reply:
Afaik, `docker-compose up` doesn't allow for custom commands to be passed into the container.